client/src/pages/AuthPages/SignUpPage/StepperForm.jsx

```diff
@@ -27,16 +27,22 @@ export default function Step1(props) {
             >
               {valueMethods.map((valueMethod) => (
                 <TextField
+                  key={valueMethod.name.toLowerCase().split(" ")[0]}
                   variant='outlined'
                   margin='normal'
-                  required
+                  required={valueMethods.length > 4 && valueMethods.length < 5 ? false : true}
```

Comment:
Need to remove length check for less than 5 since that check is useless because another component was made for confirming